caddyconfig/caddyfile/parse.go

```diff
@@ -211,10 +210,10 @@ func (p *parser) addresses() error {
 	var expectingAnother bool
 
 	for {
-		tkn := p.Val()
-
+		value := p.Val()
+		token := p.Token()
 		// special case: import directive replaces tokens during parse-time
```

Comment:
Nit: the newline was there to visually separate the sections

```suggestion
		value := p.Val()
		token := p.Token()

		// special case: import directive replaces tokens during parse-time
```